src/scss/colors.module.scss

```diff
@@ -13,7 +13,6 @@ $color-plum-200: #d1d1e8;
 $color-plum-300: #b6b7db;
 $color-plum-400: #8b8dc7;
 $color-plum-500: #6164ba;
-$color-plum-550: #5667ad;
```

Comment:
It looks like you accidentally dropped this line!

Reply:
Oops - it's [back](https://github.com/COVID19Tracking/website/pull/993/commits/8d6bfe383922759fe1f7d5f00e6ec28bf2206c6d)